apps/src/maze/api.js

```diff
@@ -312,8 +312,8 @@ exports.getPumpkin = API_FUNCTION(function (id) {
   Maze.subtype.getPumpkin(id);
 });
 
-exports.getBean = API_FUNCTION(function (id) {
-  Maze.subtype.getBean(id);
+exports.getLettuce = API_FUNCTION(function (id) {
+  Maze.subtype.getLettuce(id);
```

Comment:
I realize this is changing an existing name, but I wonder if these APIs should take the vegetable as a parameter?  E.g. `exports.harvest = API_FUNCTION(Maze.subtype)` then `harvest(LETTUCE, id)`.